test/unit/lib/plugins/aws/package/lib/merge-iam-templates.test.js

```diff
@@ -171,6 +172,43 @@ describe('lib/plugins/aws/package/lib/mergeIamTemplates.test.js', () => {
           `/aws/lambda/${service}-dev-myFunction`
         );
       });
+
+      it('should allow scheduler to assume role', () => {
```

Comment:
Let's not introduce such tests.

All tests should be configured with `runServerless`, we can confirm on added IAM rules in context of `schedule.test.js`